package.json

```diff
@@ -44,15 +44,15 @@
         "@react-native-community/datetimepicker": "^5.1.0",
         "@react-native-community/masked-view": "^0.1.11",
         "@react-native-community/netinfo": "^8.1.0",
-        "@react-native-picker/picker": "^2.3.1",
+        "@react-native-picker/picker": "^2.4.1",
         "@sentry/react-native": "^3.3.2",
         "buffer": "^6.0.3",
         "fast-text-encoding": "^1.0.3",
         "grapheme-splitter": "^1.0.4",
         "js-base64": "^3.7.2",
         "pusher-js": "^7.0.6",
         "react": "^17.0.2",
-        "react-native": "^0.67.3",
+        "react-native": "^0.68.3",

```

Comment:
Bumping react-native versions is usually a risky move. I will mark this PR as risky because of this dependency bump

Reply:
We already moved to 0.68.3. This repo has been forgotten and broken for some time so I'm bumping it to the current versions and making it work. I think you can remove risky for this case.